source/upgrade/prepare-to-upgrade-mattermost.rst

```diff
@@ -0,0 +1,187 @@
+Prepare to upgrade Mattermost
+=============================
+
+|all-plans| |self-hosted|
+
+.. |all-plans| image:: ../images/all-plans-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in Mattermost Free and Starter subscription plans.
+
+.. |self-hosted| image:: ../images/self-hosted-badge.png
+  :scale: 30
+  :target: https://mattermost.com/deploy
+  :alt: Available for Mattermost Self-Hosted deployments.
+
+In most cases, you can `upgrade Mattermost Server <https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html>`__ in a few minutes. However, the upgrade can take longer depending on several factors, including the size and complexity of your installation, and the version that you're upgrading from.
+
+.. contents::
+  :backlinks: top
+  :local:
+
+Upgrade to Mattermost v7.1
+--------------------------
+
+Mattermost v7.1 introduces schema changes in the form of a new column and its index. The following notes our test results for the schema changes:
+
+- MySQL 12M Posts, 2.5M Reactions - ~1min 34s (instance: PC with 8 cores, 16GB RAM)
+- PostgreSQL 12M Posts, 2.5M Reactions - ~1min 18s (instance: db.r5.2xlarge)
+
+Customers wanting to run the SQL queries beforehand can run the following queries before upgrading. This is fully backwards-compatible and obtains a lock on ``Reactions`` table, so users' reactions posted during this time won't be reflected in the database until the migrations are complete.
```

Comment:
```suggestion
You can run SQL queries before the upgrade to obtain a lock on ``Reactions`` table, so that users' reactions posted during this time won't be reflected in the database until the migrations are complete. This is fully backwards-compatible.
```